proto/api/read_state.proto

```diff
@@ -24,6 +24,7 @@ message ReadStakingDataMethod {
 		BUCKETS_BY_CANDIDATE = 3;
```

Comment:
let's add it after BUCKETS_BY_CANDIDATE = 3
it is gRPC proto, not data definition, no backward compatible issue